### ISSUE TITLE
PDF not rendering correctly with Chevin fonts

### ISSUE THREAD

**Issue by kennyhei** [mozilla/pdf.js#6648]
Hi,

I'm generating PDF files using Chevin fonts. They are rendered correctly in IE and Chrome but in Firefox the text is not rendered correctly. Here's my sample PDF with three lines of texts:

https://www.dropbox.com/s/r20ujvr3rlqhuqa/hello.pdf?dl=1

This is what it looks like in IE and Chrome:

![capture](https://cloud.githubusercontent.com/assets/1525463/11208893/68e995a0-8d26-11e5-85f9-6b6f16fef446.PNG)

And here's what it looks like in Firefox:

![capture2](https://cloud.githubusercontent.com/assets/1525463/11208903/82664514-8d26-11e5-8a56-93236ebf08b5.PNG)

**Comment by Snuffleupagus** [comment]
This is an issue with type CIDFontType0, and subtype `CIDFontType0C` (i.e. CFF), fonts; from the console:
```
PDF a5afcbe0e47e7d9532d15e0c290956f4 [1.4 iText® 5.5.7 ©2000-2015 iText Group NV (AGPL-version) / -] (PDF.js: 1.2.127 [WebGL]) viewer.js:6991:1
downloadable font: cmap: Range glyph reference too high (727 > 726) (font-family: "g_font_1" style:normal weight:normal stretch:normal src index:0) source: (invalid URI) <unknown>
downloadable font: rejected by sanitizer (font-family: "g_font_1" style:normal weight:normal stretch:normal src index:0) source: (invalid URI) <unknown>
downloadable font: font load failed (font-family: "g_font_1" style:normal weight:normal stretch:normal src index:0) source: (invalid URI) <unknown>
Warning: Failed to load font ""g_font_1"": SyntaxError: An invalid or illegal string was specified pdf.js:222:5
downloadable font: cmap: Range glyph reference too high (727 > 726) (font-family: "g_font_2" style:normal weight:normal stretch:normal src index:0) source: (invalid URI) <unknown>
downloadable font: rejected by sanitizer (font-family: "g_font_2" style:normal weight:normal stretch:normal src index:0) source: (invalid URI) <unknown>
downloadable font: font load failed (font-family: "g_font_2" style:normal weight:normal stretch:normal src index:0) source: (invalid URI) <unknown>
Warning: Failed to load font ""g_font_2"": SyntaxError: An invalid or illegal string was specified pdf.js:222:5
downloadable font: cmap: Range glyph reference too high (727 > 726) (font-family: "g_font_3" style:normal weight:normal stretch:normal src index:0) source: (invalid URI) <unknown>
downloadable font: rejected by sanitizer (font-family: "g_font_3" style:normal weight:normal stretch:normal src index:0) source: (invalid URI) <unknown>
downloadable font: font load failed (font-family: "g_font_3" style:normal weight:normal stretch:normal src index:0) source: (invalid URI) <unknown>
Warning: Failed to load font ""g_font_3"": SyntaxError: An invalid or illegal string was specified pdf.js:222:5
```

**Comment by Snuffleupagus** [comment]
Closing as fixed by #6715.
